build-monitor-acceptance/src/main/java/com/smartcodeltd/aether/ConsoleTransferListener.java

```diff
@@ -102,7 +102,7 @@ public void transferSucceeded(TransferEvent event) {
         TransferResource resource = event.getResource();
         long contentLength = event.getTransferredBytes();
         if (contentLength >= 0) {
-            String type = (event.getRequestType() == TransferEvent.RequestType.PUT ? "Uploaded" : "Downloaded");
+            String type = event.getRequestType() == TransferEvent.RequestType.PUT ? "Uploaded" : "Downloaded";
```

Comment:
The reason I used parenthesis here is to avoid errors with double assignment, such as:
`a = b = c ? "Uploaded" : "Downloaded"`
`a = b == c ? "Uploaded" : "Downloaded"`